module.nix

```diff
@@ -194,6 +194,18 @@ in
   };
 
   config = {
+    assertions = [
+      {
+        assertion = config.disko.imageBuilder.qemu != null -> diskoLib.vmToolsSupportsCustomQemu pkgs;
+        message = ''
+          You have set config.disko.imageBuild.qemu, but vmTools in your nixpkgs version "${lib.version}"
+          does not support overriding the qemu package with the customQemu option yet, which was added in
+          https://github.com/NixOS/nixpkgs/commit/65c851cd7523c669b8fb25236b1c48283a2f43ec
+          Please upgrade nixpkgs so that `lib.version` is at least "24.11.20240709".
+        '';
```

Comment:
Same here, I think if people need to find the exact commit it was introduced in, the files history makes it easy to do so. For an error message it's good enough to know that the current nixpkgs is too old

Reply:
Okay I agree the exact commit is not really needed for the user-facing error. I removed that part now :)